arch/arm/soc/nordic_nrf/include/nrf_common.h

```diff
@@ -61,30 +61,4 @@
 #define NRF52_IRQ_I2S_IRQn                     37
 #define NRF52_IRQ_FPU_IRQn                     38
 
-/**
- * @file UART baudrate divisors for nRFx family processors.
- *
- * Based on Nordic MDK included header file: nrf52_bitfields.h
- * Uses the UARTE_BAUDRATE macros since they are more precise.
- */
-
-#define NRF_UART_BAUDRATE_300                  0x00014000
```

Comment:
The reason that I added those is that the ones in the MDK for UART were not very accurate, so i used the ones for UARTE

Reply:
Can you verify that, with the refactor, these same values will be used?

Reply:
I do not want to mix these defines in uart_nrf5.c file. I will use UARTE defines in new shim for uarte.
In m opinion inaccuracy in UART defines is not so signifacant to cause any problem during the serial transsmition. We do not see complains in SDK releases about this.